bin/dev

```diff
@@ -6,4 +6,10 @@ then
   gem install foreman
 fi
 
-foreman start -f Procfile.dev
+if [[ "$1" == "chrome_debug" ]]; then
```

Comment:
**Question** Are we sure this value will not change in future `chrome_debug` shall we store it in an env variable instead?

Reply:
It's part of the bash script customized for the purpose of using chrome based debugger

- If we run `./bin/dev` that will run solargraph and we can connect from vscode or other supported IDE
  -  in this case that will use Procfile.dev
- If we run `./bin/dev chrome_debug` that will open up chrome for debugging.
 - In this case that will use Porcfile.chrome.dev

To just choosing, which Procfile should be use chrome_debug flag has been added

Reply:
Sounds great lets go ahead with this